exporter/collector/integrationtest/testdata/fixtures/ops_agent_host_metrics_expect.json

```diff
@@ -3150,6 +3150,486 @@
       ]
     }
   ],
+  "createMetricDescriptorRequests": [
```

Comment:
@jsuereth does this look OK for ops agent to send these descriptors?

Reply:
Ops agent can still disable CMD.  Maybe i'm misinterpreting the TODO, but I thought we just wanted to test with CMD enabled?

Reply:
Yes we want a test with it enabled.

Reply:
Oh, actually, I'm dumb.

`agent.googleapis.com` should never send metric descriptors (they're defined like platform metrics IIRC).

We want CMD turned on for any workload metrics.....  Maybe check with Ling or Sophie.

Reply:
Removed the TODO to update agent.googleapis.com tests, and added a TODO to add new integration tests for workload.googleapis.com from ops-agent